consumer/config.go

```diff
@@ -2,67 +2,53 @@ package consumer
 
 import (
 	"errors"
-	"time"
 
 	"github.com/Shopify/sarama"
-	"github.com/heetch/felice/codec"
-	"gopkg.in/retry.v1"
 )
 
 // Config is used to configure the Consumer.
+// It should be created with NewConfig before any custom
+// configuration settings are applied.
 type Config struct {
+	// Config holds the configuration used to create the consumer
+	// group instance. It must be non-nil.
 	*sarama.Config
 
-	// KafkaAddrs holds kafka brokers addresses. There must be at least
-	// one entry in the slice.
-	// The default value is "localhost:9092".
+	// KafkaAddrs holds the kafka broker addresses in host:port
```

Comment:
:tulip: Incidentally, I'd love to know if there's a proper name for "host:port" format.  I know it's defined as part of URI, but it is used outside that context so often.

Reply:
Me too. In previous places, I've tried be consistent and use "HostPort" and "Host" where appropriate, but URLs mess everything up by allowing both :)